### PR TITLE
Added pickset and avatar

### PR DESCRIPTION
**Pull request by madslundt**
Added pickset helper (http://chancejs.com/#pickset)
Added avatar web (http://chancejs.com/#avatar)

Edited pick to return T or T[]

### CODE REVIEW

File: chance/chance.d.ts [hunk 1]
```diff
@@ -115,7 +116,8 @@ declare namespace Chance {
         mixin(desc: MixinDescriptor): any;
         pad(num: number, width: number, padChar?: string): string;
         pick<T>(arr: T[]): T;
-        pick<T>(arr: T[], count: number): T[];
+        pick<T>(arr: T[], count: number): T | T[];
```
Comment (mhegazy) [inline chance/chance.d.ts]: from the documentation the return value seems to be always an array. why is it not an array?

Comment (minestarks) [inline chance/chance.d.ts]: This seems to be accurate based on the source - but been deprecated in favor of pickone() and pickset().  @madslunt can you include pickone() as well?

**Comment by paulvanbrenk** [comment]
@madslundt gentle pint

**Comment by paulvanbrenk** [comment]
Closing. This PR seems to be abandoned.

**Pull request closed without merging**